packages/webdriverio/tests/commands/browser/saveRecordingScreen.test.js

```diff
@@ -1,10 +1,10 @@
 import request from 'request'
-import { remote } from '../../../src'
```

Comment:
have to move imports after `fs` is mocked, otherwise can't mock properly `fs` imports inside commands.